src/ImmutableX.ts

```diff
@@ -839,4 +847,86 @@ export class ImmutableX {
         throw formatError(err);
       });
   }
+
+  /**
+   * Create a new Exchange transaction
+   * @param walletConnection - the pair of L1/L2 signers
```

Comment:
We may could use Eth/Stark signers here?
(As also on the all comments of the same nature inside this file)

Reply:
@RenatoChitolina done